flint.ui/src/main.js

```diff
@@ -1,21 +1,22 @@
 import Vue from "vue";
 import "es6-promise/auto";
 import Vuex from "vuex";
-
 import VueRouter from "vue-router";
 import App from "./App";
-
 import "@fortawesome/fontawesome-free/css/all.min.css";
-import "@/assets/styles/tailwind.css";
-
 import VueCompositionAPI from "@vue/composition-api"
 import store from "./store";
-
-// router setup
 import routes from "./routes/routes";
-// import VueCircleSlider from "vue-circle-slider";
 import Toast from "vue-toastification";
+import VueLayers from 'vuelayers';
+import VueSlider from 'vue-slider-component'
```

Comment:
@shubhamkarande13 indicated on #33 that you were using Vuetify for the slider component. If that is not the case we can remove Vuetify from from his PR.

Reply:
I did not try `vue-slider-component` as I was not sure about Vue 2 support. 😅
I will remove Vuetify from my PR.

Reply:
Done!